app/javascript/components/App.js

```diff
@@ -6,45 +6,65 @@ import Header from "./components/Header"
 import Footer from "./components/Footer"
 import ApptIndex from './pages/ApptIndex'
 import ApptShow from './pages/ApptShow'
-import { withRouter } from "react-router";
+import NotFound from './pages/NotFound'
 import {
   BrowserRouter as  Router,
   Route,
-  Routes,
-  
+  Switch,
 } from "react-router-dom"
 
 class App extends Component {
   constructor(props){
     super(props)
     this.state = {
-      apartments: []
+      apartments: [],
+     
     }
   }
 
   componentDidMount() {
     this.apptRead()
   }
 
+  getInfo = (id, apartment) => {
+    this.setState({ apptID: id, currentAppt: apartment})
+    console.log("get info ran");
+  }
+
   apptRead = () => {
     fetch("/apartments")
     .then(response => response.json())
     .then(apptArray => this.setState({apartments: apptArray}))
     .catch(errors => console.log(errors))
   }
+
+  updateAppt = (updatedInfo, id) => {
+    fetch("/apartments/${id}", {body: JSON.stringify(updatedInfo), 
+      headers: { "Content-Type" : "application/json"
+       },
+     method: "PATCH"
+    }).then(response => response.json())
+    .then(payload => this.apptRead())
+    .then(errors => (console.log(errors)))
+  }
   
 render(){
-  const { apartments } = this.state
+
     return (
       <Router>
         <Header {...this.props} />
-      <Routes>
-        <Route exact path="/" element={ <Home /> } />
-        <Route path="/about" element={ <AboutMe /> } />
-        <Route path="/listings"  element={<ApptIndex appts={apartments} /> } />
-        <Route path="/showlisting/:id" element={<ApptShow appts={apartments}  />} />
- 
-      </Routes>
+      <Switch>
+        <Route exact path="/" component={Home} />
+        <Route path="/about" render={ <AboutMe /> } />
```

Comment:
Looks like you went back to the older route syntax, so you'll need to update your AboutMe